x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/use_add_prebuilt_rules_table_columns.tsx

```diff
@@ -121,6 +121,7 @@ const createInstallButtonColumn = (
         disabled={isInstallButtonDisabled}
         onClick={() => installOneRule(ruleId)}
         data-test-subj={`installSinglePrebuiltRuleButton-${ruleId}`}
+        aria-label={i18n.INSTALL_RULE_BUTTON_ARIA_LABEL(record.name ?? ruleId)}
```

Comment:
In what situations do you expect rule's name to be `null` or `undefined`? According to our schemas it's always a string. On top of that would it be helpful for a use to know ruleId when installing a rule?

Reply:
Thank you, if `record.name` always defined let's use only that value.